LICENSE

```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) 2020 Aleksandr Volochnev, Aleksey Slaviansky
```

Comment:
Error, need to change Aleksei Slavianskii

Reply:
Fixed